runtime/ms-rest-azure/index.d.ts

```diff
@@ -241,15 +241,18 @@ export class BaseResource {
  *
```

Comment:
As per the handbook reference
Also note that if your main declaration file is named index.d.ts and **lives at the root of the package **(next to index.js) you do not need to mark the "types" property, _though it is advisable to do so_.